### DISCUSSION TITLE
Topics for "high level" documentation

### DISCUSSION THREAD

**Discussion by uwerat** [uwerat/qskinny#450] [Ideas]
For the first release of QSkinny we would like to have a collection of short articles to complement the more detailed doxygen documentation of the class APIs. But what are the topics we want to cover as articles ?

**Comment by uwerat** [comment]
"QSkinny Box Model"

The vast majority of a user interface is made from 3 basic elements:

- image
- text
- box

QSkinny uses a specific model to describe boxes - similar to the one used for Qt style sheets: https://doc.qt.io/qt-6/stylesheet-customizing.html

**Comment by uwerat** [comment]
"Qskinny Layouts"

The layout system of QSkinny is similar to the one of Qt/Widgets - more powerful than the declarative system of Qt/Quick. However using it in a proper way requires a certain understanding of how the informations provided by control and layout play together.

**Reply by peter-ha** [reply]
We already have something about layouts (https://qskinny.github.io/docs/tutorials/04-Layouts/); I guess we could just go into more depth?

Also a basic understanding of skins, (local and global) skin hints, and skinlets would be another topic I guess?
